src/app/donation-complete/donation-complete.component.ts

```diff
@@ -21,20 +25,24 @@ export class DonationCompleteComponent {
   public donation: Donation;
   public giftAidAmount: number;
   public noAccess = false;
+  offerToSetPassword = false;
```

Comment:
@NoelLH Is there a reason you've omitted the access modifier here?

Reply:
@alihejazi Yes, `public` is the default and I have worked with TypeScript (and I think Angular) style guides before that explicitly suggest it is good practice not to include it for readability. Angular's docs' own examples also leave it out when demonstrating properties to be shared with templates. So I tend these days to phase it out as I go. I don't think it's worth refactoring the whole app but open to suggestions.